lib/heuristics/scheduling_heuristic.rb

```diff
@@ -842,8 +842,13 @@ def compatible_days(service_id, day)
       !@services_data[service_id][:raw].unavailable_days.include?(day)
     end
 
+    def compatible_vehicle(service_id, route_data)
+      route_data[:vehicle].skills.any?{ |skill_set| (@services_data[service_id][:raw].skills - skill_set).empty? }
+    end
```

Comment:
Like this we risk two incompatible services get assigned to the same vehicle because one needs first skill_set, the other needs the second skill_set but they return true. However, skills_sets are not cumulatif (if I am not mistaken). 

Either we need a way to distinguish which skills left "eliminated" or we need to add an assert so that periodic heuristic can handle at most one skill_set.

It is not hard to achieve the first but it is a bit delicate (if the user doesn't expect to have multiple skill_sets, I suggest we add a note here and an assert to the wrapper.rb)

Reply:
I agree with this suggestion :) especially because alternative skills are not needed in periodic_heuristic yet. But thank you for noticing, I do not think you are mistaken.